assets/src/edit-story/output/page.js

```diff
@@ -119,13 +119,15 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
         </amp-story-grid-layer>
       </StoryAnimation.Provider>
       {hasPageAttachment && (
-        <amp-story-page-attachment
+        <amp-story-page-outlink
           layout="nodisplay"
-          href={page.pageAttachment.url}
-          data-cta-text={
-            page.pageAttachment.ctaText || __('Learn more', 'web-stories')
-          }
-        />
+          cta-image={page.pageAttachment.icon}
+          theme={page.pageAttachment.theme}
```

Comment:
Without a migration, this is just left empty for old stories - I assume that's fine?

Reply:
Might want to check whether it doesn't cause AMP validation errors if omitted or an empty string.

A data migration would remove any uncertainty here.

Reply:
The default is the light theme so it visibly at least works as expected for the old stories, didn't check for validation errors though.

We can add the migration to be certain.